.coderabbit.yml

```
@@ -0,0 +1,112 @@
+  - path: "**/tests/**/*.php"
+    instructions: |
+      Ensure comprehensive test coverage, proper test naming, and follow PHPUnit best practices.
+  - path: "**/database/migrations/*.php"
+    instructions: |
+      Ensure proper foreign key constraints, indexes, and follow Laravel migration best practices.
+  - path: "**/routes/*.php"
+    instructions: |
+      Ensure proper route naming, middleware usage, and API versioning if applicable.
+  - path: "**/config/*.php"
+    instructions: |
+      Ensure proper configuration structure and documentation of config options.
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)